src/Pages/AdminHomePage/AdminHomePage.tsx

```diff
@@ -237,22 +206,25 @@ const AdminHomePage = () => {
             />
           </div>
         </div>
-        <div className='w-[100%] bg-[#FFFFFF] border rounded-t-lg border-[#E2E4EB] rounded-b-lg hidden xl:order-last lg:order-last order-first md:block xl:col-span-2 col-span-1'>
+        <div className='w-[100%] h-[1057px] bg-[#FFFFFF] border rounded-t-lg border-[#E2E4EB] rounded-b-lg hidden xl:order-last lg:order-last order-first md:block xl:col-span-2 col-span-1'>
           <div className='h-[48px] bg-[#F6F8FA] flex items-center p-[10px] justify-between'>
-            <div className='text-[#828DA9] text-[20px] font-[400]'>Recent Activity</div>
+            <div className='text-[#828DA9] text-[20px] font-[400]'>Available Tasks</div>
             <div className='border w-[32px] h-[32px] flex items-center justify-center rounded-full'>
               <ArrowOutwardOutlined color="disabled" style={{ fontSize: 'medium' }} />
             </div>
           </div>
-          <div className='w-[100%] p-[10px] pt-[0px] '>
-            {activities.map((activity, index) => (
-              <ActivityLogCard
-                key={index}
-                title={activity.title}
-                text={activity.text}
-                dateTime={activity.dateTime}
-              />
-            ))}
+          <div className='h-[1000px] overflow-y-auto'>
+            <div className='w-[100%] p-[10px] pt-[0px] '>
+              {isSuccess && Array.isArray(data?.data) && data.data.map((activity: any, index: number) => {
+                return (
```

Comment:
where are the links ? 
on clicking each card, should take us to the customer or supplier page